### ISSUE TITLE
Feature request - Display how long an issue has been in a pipeline

### ISSUE THREAD

**Issue by andypiccolo** [ZenHubIO/support#821]
This should be an easy thing to do and would be very helpful.  My team operates in Kanban and we encourage tasks to be broken up into week or less durations.  It would be very helpful for everyone to see especially during team reviews of the boards how long each issue has been in in progress.

Thanks!
Andy

**Comment by azenMatt** [comment]
Hi @andypiccolo - we are looking at releasing better ordering for pipelines - aka sort contents by creation date, age, recently updated etc... 

Would that meet your use case here?

**Comment by andypiccolo** [comment]
Yes if you included age, essentially when it was dragged into that pipeline that would be perfect.


My team works Kanban and we try to size issues to be a week or less in duration if possible.  Having the age of an issue would provide good feedback to the team.  Right now I have a homegrown version of it that I wrote using the API, it sucks but it works.  I'd love to have it be part of the product proper.  Sometime people don't realize how long things have been sitting there in a particular pipeline.  Especially in the backlogs.


Appreciate you reaching out.  We really do find your product quite helpful, we are up to about 30 users now.


Cheers,

Andy


________________________________

**Comment by azenMatt** [comment]
sounds good @andypiccolo - we are looking at bringing in something like...

![image](https://cloud.githubusercontent.com/assets/7584089/17943291/a60c49c2-69ef-11e6-883c-c132a598ad0c.png)

In the next couple of releases, I have added to the scope of this to include a 'Time in Pipeline' ordering option (open to better ideas on the name!) which will then allow you to separate out and see what has been here for a while

I will be able to confirm tomorrow if this is in the next sprint or not

**Comment by andypiccolo** [comment]
Matt,


Awesome, that will do what I am looking for.

Andy

________________________________
